internal/common/objects/BaseBiker.go

```diff
@@ -0,0 +1,114 @@
+package basebiker
+
+import (
+	utils "SOMAS2023/internal/common/utils"
+
+	"math/rand"
+
+	baseAgent "github.com/MattSScott/basePlatformSOMAS/BaseAgent"
+	"github.com/google/uuid"
+)
+
+// placeholder
+type ServerStatus struct {
+	// []lootboxes
+	// []bikes
+	// []bikers
+}
+
```

Comment:
This would be more of a GameState than ServerStatus. Also, since we're dealing with constant addition/removal of agents/lootboxes, a hashmap would be more efficient